common/containers/Tabs/SendTransaction/index.jsx

```diff
@@ -238,8 +268,63 @@ export class SendTransaction extends React.Component {
     return { to, data, value, unit, gasLimit, readOnly };
   }
 
+  isValid() {
+    const { to, value } = this.state;
+    return (
+      isValidETHAddress(to) &&
+      value &&
+      Number(value) > 0 &&
+      !isNaN(Number(value)) &&
+      isFinite(Number(value))
+    );
+  }
+
+  // FIXME MOVE ME
+  getTransactionFromState(): ?TransactionWithoutGas {
```

Comment:
I think I would also like to figure that out as part of #100, I'm pretty unsure of how the two will fit together at the moment, and how much of the transaction generation code will be component specific vs reusable. I'd rather not prematurely make that call and have to refactor twice.